mongoose/messages/group-schema.ts

```diff
@@ -0,0 +1,22 @@
+/**
+ * @file Implements mongoose schema
+ * for the groups collection
+ */
+
+import mongoose, {Schema} from "mongoose";
+import Group from "../../models/messages/group";
+
+const UserSchema = new Schema({ name: String });
+
+
+const GroupSchema = new mongoose.Schema<Group>({
+    name: {type: String, required: true},
+    users: [UserSchema],
```

Comment:
I think this should be `users: [{type: Schema.Types.ObjectId, ref: 'UserModel'}]`